### ISSUE TITLE
batch normalization 추가

### ISSUE THREAD

**Issue by joongsoo** [JSDanielPark/dqn_bitcoin_auto_trading#17]
- batch normalization 추가
- learning rate 높이기
